module/commands/help.py

```diff
@@ -37,32 +38,23 @@ def help_cmd(update: Update, context: CallbackContext):
     ])
 
     reply_markup = InlineKeyboardMarkup(keyboard)
+    if(edit):
+        context.bot.editMessageText(text=message_text, chat_id=update.message.chat_id, message_id=update.message.message_id, reply_markup=reply_markup)
+    else:
+        context.bot.sendMessage(chat_id=chat_id, text=message_text, reply_markup=reply_markup)
+    
 
-    context.bot.sendMessage(chat_id=chat_id, text=message_text, reply_markup=reply_markup)
-
-def rapp_menu(update: Update, context: CallbackContext, chat_id: int, message_id: int):
-    """Called by the sm_rapp_menu button from the /help command.
-    Allows the user to select the department
+def help_back_to_menu(update: Update, context: CallbackContext, chat_id: int, message_id: int):
+    """Called by a sm_help_back_to_menu button from the sub menu.
+    Allows the user to return back to the command list
 
     Args:
         update: update event
         context: context passed by the handler
         chat_id: id of the chat the command was invoked from
         message_id: id of the help message
     """
-    message_text = "Quali rappresentanti vuoi contattare?"
-
-    keyboard = [[]]
-    keyboard.append(
-        [
-            InlineKeyboardButton("Rapp. DMI",         callback_data="md_rappresentanti_dmi"),
-            InlineKeyboardButton("Rapp. Informatica", callback_data="md_rappresentanti_informatica"),
-            InlineKeyboardButton("Rapp. Matematica",  callback_data="md_rappresentanti_matematica"),
-        ]
-    )
-    reply_markup = InlineKeyboardMarkup(keyboard)
-
-    context.bot.editMessageText(text=message_text, chat_id=chat_id, message_id=message_id, reply_markup=reply_markup)
+    help_cmd(update, context, chat_id=chat_id, message_id=message_id)
```

Comment:
In this case, you should pass
```python
help_cmd(update, context, edit=True)
```

Reply:
My bad, I pushed to fast without double checking the code flow.